docs/topics/con_about-shared-messaging-work-queue.adoc

```diff
@@ -0,0 +1,12 @@
+[id='about-circuit-breaker_{context}']
```

Comment:
Not circuit breaker :-)

Reply:
Whoah! thanks for catching that. Definitely not supposed to be there :) Fixing...